README.md

```diff
@@ -1,428 +1,240 @@
-print (wallet)
-"Wallet (default, default, ~/.bittensor/wallets/)"
-```
-2. Or using btcli
-> Use the subcommand `wallet` or it's alias `w`:
-```bash
-$ btcli wallet new_coldkey
-    Enter wallet name (default):      
-
-    IMPORTANT: Store this mnemonic in a secure (preferably offline place), as anyone who has possession of this mnemonic can use it to regenerate the key and access your tokens. 
-    The mnemonic to the new coldkey is:
-    **** *** **** **** ***** **** *** **** **** **** ***** *****
-    You can use the mnemonic to recreate the key in case it gets lost. The command to use to regenerate the key using this mnemonic is:
-    btcli w regen_coldkey --mnemonic post maid erode shy captain verify scan shoulder brisk mountain pelican elbow
-
-$ btcli wallet new_hotkey
-    Enter wallet name (default): d1
-    Enter hotkey name (default): 
-
-    IMPORTANT: Store this mnemonic in a secure (preferably offline place), as anyone who has possession of this mnemonic can use it to regenerate the key and access your tokens. 
-    The mnemonic to the new hotkey is:
-    **** *** **** **** ***** **** *** **** **** **** ***** *****
-    You can use the mnemonic to recreate the key in case it gets lost. The command to use to regenerate the key using this mnemonic is:
-    btcli w regen_hotkey --mnemonic total steak hour bird hedgehog trim timber can friend dry worry text
-```
-In both cases you should be able to view your keys by navigating to ~/.bittensor/wallets or viewed by running ```btcli wallet list```
-```bash
-$ tree ~/.bittensor/
-    .bittensor/                 # Bittensor, root directory.
-        wallets/                # The folder containing all bittensor wallets.
-            default/            # The name of your wallet, "default"
-                coldkey         # You encrypted coldkey.
-                coldkeypub.txt  # Your coldkey public address
-                hotkeys/        # The folder containing all of your hotkeys.
-                    default     # You unencrypted hotkey information.
-```
-Your default wallet ```Wallet (default, default, ~/.bittensor/wallets/)``` is always used unless you specify otherwise. Be sure to store your mnemonics safely. If you lose your password to your wallet, or the access to the machine where the wallet is stored, you can always regenerate the coldkey using the mnemonic you saved from above. 
-```bash
-$ btcli wallet regen_coldkey --mnemonic **** *** **** **** ***** **** *** **** **** **** ***** *****
-```
+In Bittensor there is one blockchain, and many platforms that are connected to this one blockchain. We call these platforms as **subnets**, and this one blockchain as **subtensor**. So, a subnet can be AI-related or it can be something else. The Bittensor network has a number of distinct subnets. All these subnets interact with subtensor blockchain. If you are thinking, "So, subnets are not part of the blockchain but only interact with it?" then the answer is "yes, exactly."
```

Comment:
```suggestion
In Bittensor there is one blockchain, and many platforms that are connected to this one blockchain. We call these platforms as **subnets**, and this one blockchain  **subtensor**. So, a subnet can be AI-related or it can be something else. The Bittensor network has a number of distinct subnets. All these subnets interact with subtensor blockchain. If you are thinking, "So, subnets are not part of the blockchain but only interact with it?" then the answer is "yes, exactly."
```

removed the "as" to be more grammatically correct (I think)